plugins/modules/clustering/nomad/nomad_job.py

```diff
@@ -0,0 +1,252 @@
+      type: int
+      default: 5
+    validate_certs:
+      description:
+        - Enable TLS/SSL certificate validation.
+      type: bool
+      default: true
+    client_cert:
+      description:
+        - Path of certificate for TLS/SSL.
+      type: path
+    client_key:
+      description:
+        - Path of certificate's private key for TLS/SSL.
+      type: path
+    namespace:
+      description:
+        - Namespace for Nomad.
+      type: str
+    token:
+      description:
+        - ACL token for authentification.
+      type: str
+    name:
+      description:
+        - Name of job for delete,stop and start job without source.
+        - Name of job for delete, stop and start job without source.
+        - Either this or I(content) must be specified.
+      type: str
+    state:
+      description:
+        - Deploy or remove job.
+      choices: ["present", "absent"]
+      required: true
+      type: str
+    force_start:
+      description:
+        - Force job to started.
+      type: bool
+      default: false
+    content:
+      description:
+        - Content of Nomad job.
+        - Either this or I(name) must be specified.
+      type: str
+    content_format:
+      description:
+        - Type of content of Nomad job.
+      choices: ["hcl", "json"]
+      default: hcl
+      type: str
+'''
+
+EXAMPLES = '''
+- name: Create job
+  community.general.nomad_job:
+    host: localhost
+    state: present
+    content: "{{ lookup('ansible.builtin.file', 'job.hcl') }}"
+    timeout: 120
+- name: Stop job
+  community.general.nomad_job:
+    host: localhost
+    state: absent
+    name: api
+- name: Force job to start
+  community.general.nomad_job:
+    host: localhost
+    state: present
+    name: api
+    timeout: 120
+    force_start: true
+'''
+
+
+import os
+import json
+
+from ansible.module_utils.basic import AnsibleModule, missing_required_lib
+from ansible.module_utils._text import to_native
+
+import_nomad = None
+try:
+    import nomad
+    import_nomad = True
+except ImportError:
+    import_nomad = False
+
+
+def run():
+    module = AnsibleModule(
+        argument_spec=dict(
+            host=dict(required=True, type='str'),
+            state=dict(required=True, choices=['present', 'absent']),
+            use_ssl=dict(type='bool', default=False),
+            timeout=dict(type='int', default=5),
+            validate_certs=dict(type='bool', default=True),
+            client_cert=dict(type='path', default=None),
+            client_key=dict(type='path', default=None),
+            namespace=dict(type='str', default=None),
+            name=dict(type='str', default=None),
+            content_format=dict(choices=['hcl', 'json'], default='hcl'),
+            content=dict(type='str', default=None),
+            force_start=dict(type='bool', default=False),
+            token=dict(type='str', default=None, no_log=True)
+        ),
+        mutually_exclusive=[
```

Comment:
```suggestion
        ),
        supports_check_mode=?,
        mutually_exclusive=[
```
after deciding True or False (and implementing if True), please, put info about it in `note:` doc section.

Reply:
ok it's added